include/fiction/algorithms/simulation/sidb/is_ground_state.hpp

```diff
@@ -5,45 +5,66 @@
 #ifndef FICTION_IS_GROUND_STATE_HPP
 #define FICTION_IS_GROUND_STATE_HPP
 
-#include "fiction/algorithms/simulation/sidb/minimum_energy.hpp"
-#include "fiction/algorithms/simulation/sidb/quickexact.hpp"
-#include "fiction/algorithms/simulation/sidb/quicksim.hpp"
+#include "fiction/algorithms/simulation/sidb/determine_groundstate_from_simulation_results.hpp"
 #include "fiction/algorithms/simulation/sidb/sidb_simulation_result.hpp"
 #include "fiction/traits.hpp"
-#include "fiction/utils/math_utils.hpp"
 
-#include <cmath>
+#include <cstdint>
+#include <unordered_set>
 
 namespace fiction
 {
 
 /**
- * This function checks if the ground state is found by the *QuickSim* algorithm.
+ * This function checks if the ground state is found by the heuristic.
  *
  * @tparam Lyt Cell-level layout type.
  * @param heuristic_results All found physically valid charge distribution surfaces obtained by a heuristic algorithm.
- * @param exhaustive_results All valid charge distribution surfaces determined by ExGS.
+ * @param exact_results All valid charge distribution surfaces determined by ExGS.
```

Comment:
The docstring mentions physically valid layouts while the data type is a `sidb_simulation_result`. I know it internally contains the CDSs. Can we however adjust this docstring such that the user doesn't need to worry about what's included in the `sidb_simulation_result` and just passes whatever they obtained from their simulation algorithm?

Reply:
sorry, this was just wrong.